### PR TITLE
INFRA-967 target_group alarms

### PR DESCRIPTION
**Pull request by sannithibalaji**

### CODE REVIEW

File: cloudlift/deployment/service_template_generator.py [hunk 1]
```diff
@@ -949,31 +959,60 @@ def _add_alb_alarms(self, service_name, alb):
...
+                AlarmActions=[Ref(self.notification_sns_arn)],
+                OKActions=[Ref(self.notification_sns_arn)],
+                AlarmDescription=f'Triggers if {elb_5xx_error_code} response originated from load balancer',
+                Namespace='AWS/ApplicationELB',
+                Period=60,
+                ComparisonOperator='GreaterThanOrEqualToThreshold',
+                Statistic='Sum',
+                Threshold=threshold,
+                MetricName=f'HTTPCode_ELB_{elb_5xx_error_code}_Count',
+                TreatMissingData='notBreaching'
```
Comment (aswinkarthik) [inline cloudlift/deployment/service_template_generator.py:1011]: TreatMissingData should be `good` right? I have noticed that this wont resolve the pagerduty on its own if 5xx stops coming

File: test/deployment/service_template_generator_test.py [hunk 2]
```diff
@@ -244,8 +245,11 @@ def mock_build_config_impl(env_name, cloudlift_service_name, sample_env_file_pat
         generated_template = template_generator.generate_service()
         template_file_path = os.path.join(os.path.dirname(__file__),
                                           '../templates/expected_service_with_new_alb_template.yml')
-        with(open(template_file_path)) as expected_template_file:
-            assert to_json(generated_template) == to_json(''.join(expected_template_file.readlines()))
+        with(open(template_file_path, 'w')) as expected_template_file:
```
Comment (aswinkarthik) [inline test/deployment/service_template_generator_test.py:248]: i think you forgot to revert this part

**Review by aswinkarthik — CHANGES_REQUESTED** [review]

File: test/deployment/service_template_generator_test.py (hunk 2 above)
Comment (sannithibalaji) [reply test/deployment/service_template_generator_test.py:248]: yes, thank you. 
will revert it.

File: cloudlift/deployment/service_template_generator.py (hunk 1 above)
Comment (sannithibalaji) [reply cloudlift/deployment/service_template_generator.py:1011]: Valid values are breaching, notBreaching, ignore, and missing.

Comment (sannithibalaji) [reply cloudlift/deployment/service_template_generator.py:1011]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cw-alarm.html#cfn-cloudwatch-alarms-treatmissingdata

File: test/deployment/service_template_generator_test.py (hunk 2 above)
Comment (sannithibalaji) [reply test/deployment/service_template_generator_test.py]: reverted it.

**Review by aswinkarthik — APPROVED** [review]

**Pull request merged**